src/Makefile.am

```diff
@@ -3,7 +3,7 @@ LDADD = $(LIBINTL)
 
 lib_LTLIBRARIES = libbytesize.la
 libbytesize_la_CFLAGS = -Wall -Wextra -Werror -Wno-overflow -D_GNU_SOURCE
-libbytesize_la_LIBADD = -lgmp -lmpfr -lpcre
+libbytesize_la_LIBADD = -lgmp -lmpfr -lpcre2-8
```

Comment:
Can we use `$PCRE2-LIBS` or something like that here?

Reply:
Fixed.